src/views/penalty-details.njk

```diff
@@ -54,6 +55,21 @@
             autocomplete: "off"
           }) 
         }}
+        <div class="govuk-form-group">
+          {% set detailsHTML %}
+          <p>
+            Your company number and reference number are shown at the bottom of your penalty notice.
+          </p>
+          <img src="{{ ROOT_URI }}/images/penalty-notice-image.png" class="call-out-image" alt=""/>
```

Comment:
Being picky a bit here but I do not see much value in keeping word image in the image name :-)

Reply:
i've changed this